src/onnx/parse_scatter.cpp

```diff
@@ -0,0 +1,43 @@
+#include <migraphx/onnx/op_parser.hpp>
+#include <migraphx/instruction.hpp>
+#include <migraphx/ranges.hpp>
+#include <migraphx/make_op.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace onnx {
+
+struct parse_scatter : op_parser<parse_scatter>
+{
+    std::vector<op_desc> operators() const { return {{"ScatterElements"}}; }
+
+    instruction_ref parse(const op_desc& /*opd*/,
+                          const onnx_parser& /*parser*/,
+                          const onnx_parser::node_info& info,
+                          const std::vector<instruction_ref>& args) const
+    {
+        operation op;
+
+        std::string op_name = "scatter_none";
```

Comment:
I would prefer to add another string variable called `reduction_mode` and default it to "none." Then change `op_name` to "scatter_". In L36 where `make_op` is being called you can do something like `op_name + reduction_mode`.

Reply:
If it's just a matter of style, I prefer to keep this the way it is.  I think it's easier to follow and less prone to errors in the future.